src/Dofs/DofHandler.jl

```diff
@@ -807,13 +807,11 @@ function _evaluate_at_grid_nodes(dh::DofHandler, u::Vector{T}, fieldname::Symbol
         local_node_coords = reference_coordinates(ip_geo)
         qr = QuadratureRule{getdim(ip), getrefshape(ip)}(zeros(length(local_node_coords)), local_node_coords)
         ip = getfieldinterpolation(fh, field_idx)
-        if ip isa ScalarInterpolation
-            cv = CellScalarValues(qr, ip, ip_geo)
-        elseif ip isa VectorizedInterpolation
+        if ip isa VectorizedInterpolation
             # TODO: Remove this hack when embedding works...
-            cv = CellScalarValues(qr, ip.ip, ip_geo)
+            cv = CellValues(qr, ip.ip, ip_geo)
         else
-            cv = CellVectorValues(qr, ip, ip_geo)
+            cv = CellValues(qr, ip, ip_geo)
```

Comment:
This is because we can't currently construct CellValues for embedded elements (computing the gradient fails). We can add a new CellValue subtype that can only be used for evalutation perhaps.